### ISSUE TITLE
add key with loaded file name

### ISSUE THREAD

**Issue by jscotka** [psss/fmf#11]
Actual format of FMF output lost info about FMF config file location, what will have more usages:
 *  **debugging** - able  to see from which file data cames from
 * **test path location** - help to find where is the ``test:`` key

like: ``main.fmf``
```
/a/1:
  test: 1
/a/2:
  test: 2
```
and ``second.fmf``
```
/b/1:
  test: 1
```
produces:
```
$ fmf
tmp/second/b/1
test: 1
tmp/a/2
test: 2
tmp/a/1
test: 1
```

would be:
```
$ fmf
fmf_invocation_path: /home/jscotka/tmp
tmp/second/b/1
test: 1
fmf_metadata_file: second.fmf
tmp/a/2
test: 2
fmf_metadata_file: main.fmf
tmp/a/1
test: 1
fmf_metadata_file: main.fmf
```
just little question is how to deal with merges? does that make sense to have there list of files, instead of one file string, or that there is most relevant (latest) is enough and sufficient?
     Honza

**Comment by jscotka** [comment]
done by: https://github.com/psss/fmf/pull/15

**Comment by psss** [comment]
This has been implemented by 7486060d76b3564d3e83cf8ed5e9e7b3df6c62d7.
